.env-devel

```diff
@@ -235,6 +235,7 @@ WEBSERVER_GARBAGE_COLLECTOR=null
 WEBSERVER_DB_LISTENER=0
 
 WEBSERVER_LOGIN={}
+LOGIN_ACCOUNT_DELETION_RETENTION_DAYS=31
```

Comment:
you probably need a MR in osparc-configs

Reply:
The link to that MR is already in the `devops` section of this PR :-)